### PR TITLE
solved: day02

### PR DESCRIPTION
**Pull request by dalprahcd**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new benchmarking entry for "Day 2" alongside adjustments to "Day 1" benchmarking times.
	- Added game data analysis functionality involving cubes of different colors and quantities.

- **Documentation**
	- Updated README to reflect new benchmarking entries and adjustments.

- **Data**
	- Added a new data file for representing game results with colored items distribution.

### CODE REVIEW

File: src/bin/02.rs [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        Round::new(vec![Cubes::new(2, "green")]),
+    ]), 48)]
+    #[case(Game::new(2, vec![
+        Round::new(vec![Cubes::new(1, "blue"), Cubes::new(2, "green")]),
+        Round::new(vec![Cubes::new(3, "green"), Cubes::new(4, "blue"), Cubes::new(1, "red")]),
+        Round::new(vec![Cubes::new(1, "green"), Cubes::new(1, "blue")]),
+    ]), 12)]
+    fn test_game_power(#[case] game: Game, #[case] expected: u32) {
+        assert_eq!(expected, game.power())
+    }
```
[the remark this answers is not in the document]
Comment (dalprahcd) [reply src/bin/02.rs:227]: @coderabbitai Yes, can you generate additional tests for edge cases or invalid inputs?
